lib/nerves_network/dhcp_manager.ex

```diff
@@ -162,7 +167,7 @@ defmodule Nerves.Network.DHCPManager do
     Logger.debug("(context = :down) :ifdown info: #{inspect(info)} state = #{inspect(state)}")
 
     state
```

Comment:
shouldnt "state" be removed here ? we're not piping it into stop client

Reply:
We do as first parameter